core/class/GeoPolitica.php

```diff
@@ -387,5 +387,73 @@ public function gruppi() {
         ], 'nome ASC');
         return $g;
     }
+
+    /**
+     * Cancella Geopolitica
+     * @param GeoPolitica
+     */
+    public function cancella(){
+
+        $oid = $this->oid();
+
+        /* Cancello autoparchi e veicoli ad esso associati li passo al nazionale */
+        $autoparchi = Autoparco::filtra([
+          ['comitato', $oid]
+        ]);
+        foreach($autoparchi as $autoparco){
+            $collocazioni = Collocazione::filtra([['autoparco', $autoparco]]);
+
+            foreach ( $collocazioni as $collocazione ){
+                $collocazione->cancella();
+            }
+
+            $auoparco->cancella();
+        }
+
+        /* Cancello i corsi base */
+        $corsibase = CorsoBase::filtra([
+          ['comitato', $oid]
+        ]);
+        foreach($corsibase as $corsobase){
+            $lezioni = Lezione::filtra([['corso', $corsobase]]);
+            foreach( $lezioni as $lezione ){
+                $assenze = AssenzaLezione::filtra([['lezione', $lezione]]);
```

Comment:
... e, per lo stesso principio, cancellare le assenze dovrebbe essere fatto in `Lezione->cancella()`